components/script/dom/servoparser/mod.rs

```diff
@@ -91,6 +92,7 @@ pub struct ServoParser {
     aborted: Cell<bool>,
     /// https://html.spec.whatwg.org/multipage/#script-created-parser
     script_created_parser: bool,
+    async: bool,
```

Comment:
You don't need this, the provided tokeniser already lets you know whether it's async or not, please remove.